damus/Util/Hashtags.swift

```diff
@@ -9,60 +9,70 @@ import Foundation
-    static let plebchain = CustomHashtag(name: "plebchain", color: DamusColors.deepPurple, offset: -3.0)
-    static let zap = CustomHashtag(name: "zap", color: DamusColors.yellow, offset: -4.0)
 }
 
-
 let custom_hashtags: [String: CustomHashtag] = [
-    "bitcoin": CustomHashtag.bitcoin,
-    "btc": CustomHashtag.bitcoin,
-    "nostr": CustomHashtag.nostr,
-    "coffee": CustomHashtag.coffee,
-    "coffeechain": CustomHashtag.coffee,
-    "plebchain": CustomHashtag.plebchain,
-    "zap": CustomHashtag.zap,
-    "zaps": CustomHashtag.zap,
-    "zapathon": CustomHashtag.zap,
-    "onlyzaps": CustomHashtag.zap,
+    "bitcoin": CustomHashtag(imageName: "bitcoin", display: "Bitcoin", color: Color.orange, offset: -3.0),
+    "bitcoinwallet": CustomHashtag(imageName: "bitcoin", display: "Bitcoin", color: Color.orange, offset: -3.0),
+    "bitcoinadoption": CustomHashtag(imageName: "bitcoin", display: "BitcoinAdoption", color: Color.orange, offset: -3.0),
+    "bitcoinjungle": CustomHashtag(imageName: "bitcoin", display: "BitcoinJungle", color: Color.orange, offset: -3.0),
+    "btc": CustomHashtag(imageName: "bitcoin", display: "BTC", color: Color.orange, offset: -3.0),
+    "coffee": CustomHashtag(imageName: "coffee", display: "Coffee", color: DamusColors.brown, offset: -1.0),
+    "coffeechain": CustomHashtag(imageName: "coffee", display: "CoffeeChain", color: DamusColors.brown, offset: -1.0),
+    "nostr": CustomHashtag(imageName: "nostr", display: "Nostr", color: DamusColors.purple, offset: -2.0),
+    "nostrica": CustomHashtag(imageName: "nostr", display: "Nostrica", color: DamusColors.purple, offset: -2.0),
+    "onlyzap": CustomHashtag(imageName: "zap", display: "OnlyZap", color: DamusColors.yellow, offset: -4.0),
+    "onlyzaps": CustomHashtag(imageName: "zap", display: "OnlyZaps", color: DamusColors.yellow, offset: -4.0),
+    "plebchain": CustomHashtag(imageName: "plebchain", display: "PlebChain", color: DamusColors.deepPurple, offset: -3.0),
+    "zap": CustomHashtag(imageName: "zap", display: "Zap", color: DamusColors.yellow, offset: -4.0),
+    "zaps": CustomHashtag(imageName: "zap", display: "Zaps", color: DamusColors.yellow, offset: -4.0),
+    "zapathon": CustomHashtag(imageName: "zap", display: "Zapathon", color: DamusColors.yellow, offset: -4.0),
 ]
 
 func hashtag_str(_ htag: String) -> CompatibleText {
-    var attributedString = AttributedString(stringLiteral: "#\(htag)")
-    attributedString.link = URL(string: "damus:t:\(htag)")
-    
     let lowertag = htag.lowercased()
     
-    var text = Text(attributedString)
+    let displayText: String
+    var displayColor: Color?
+    var imageOffset: CGFloat?
+    var imageName: String?
+    
     if let custom_hashtag = custom_hashtags[lowertag] {
-        if let col = custom_hashtag.color {
-            attributedString.foregroundColor = col
-        }
-        
-        let name = custom_hashtag.name
+        displayText = custom_hashtag.display
```

Comment:
let's make this an accessibility option, only camelCase it if it's on (default off)

Reply:
Done. NB Setting applies to only new posts.

Updated default to false.